tests/foreman/api/test_repositories.py

```diff
@@ -17,11 +17,14 @@
 :Upstream: No
 """
 import pytest
+from manifester import Manifester
 from nailgun import entities
 from nailgun.entity_mixins import call_entity_method_with_timeout
 from requests.exceptions import HTTPError
 
 from robottelo import constants
+from robottelo.cli.base import CLIReturnCodeError
+from robottelo.cli.subscription import Subscription
```

Comment:
These are need not to be imported now simply do :

```suggestion
target_sat.cli.Subscription
```